src/psake.psd1

```diff
@@ -1,6 +1,6 @@
 @{
     RootModule        = 'psake.psm1'
-    ModuleVersion     = '4.6.0'
+    ModuleVersion     = '4.7.0'
     GUID              = 'cfb53216-072f-4a46-8975-ff7e6bda05a5'
     Author            = 'James Kovacs'
     Copyright         = 'Copyright (c) 2012-16 James Kovacs, Damian Hickey and Contributors'

```

Comment:
@devblackops actually, thinking about this, shouldn't the Copyright be extended to 2017?

Reply:
Good catch. I’ll update it.